### ISSUE TITLE
[Misc]: More patch versions

### ISSUE THREAD

**Issue by mikelfried** [vllm-project/vllm#9200]
### Anything you want to discuss about vllm.

Hi team,

It seems that vllm version 0.6.2 is having some issues when used with models like Qwen-VL-2, Pixtral, and others (#9068, #9091...).
**Most of the bugs have already been fixed in the current dev branch**, but they haven’t been included in a new version yet, which makes it a bit tricky to use in a production environment.

I'm really grateful for all the hard work you're putting into this project! It would be awesome if there could be more frequent version releases, especially patch versions that address bug fixes.

Thanks a lot!
Michael

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by simon-mo** [comment]
We aim to have release every two weeks but sometimes blocked by major features or model releases. We believe our nightly versions should be fairly stable, which that suffice?

**Comment by mikelfried** [comment]
Oh, I just noticed the release of the nightly versions—thanks for that! I suppose there aren’t any nightly Docker images available, right?

**Comment by russellb** [comment]
> Oh, I just noticed the release of the nightly versions—thanks for that! I suppose there aren’t any nightly Docker images available, right?

right - I haven't seen any

**Comment by ycool** [comment]
> Oh, I just noticed the release of the nightly versions—thanks for that! I suppose there aren’t any nightly Docker images available, right?

```
export VLLM_COMMIT=07c11cf4d4b9a913fa52142fe134849f1e25e393 # use full commit hash from the main branch
docker pull public.ecr.aws/q9t5s3a7/vllm-ci-test-repo:${VLLM_COMMIT}
```

**Comment by mikelfried** [comment]
@ycool Thank you so much! Really appreciate it!
